app/api/controllers/RequestHandler.scala

```diff
@@ -123,21 +126,34 @@ object RequestHandler {
 
       implicit class Response(result: Result) {
 
+        private val HttpDateFormatter = DateTimeFormatter
```

Comment:
If this is for IMF date format would it be more descriptive to name it something like
```suggestion
        private val IMFDateFormatter = DateTimeFormatter
```

Reply:
also, lower case H or I in either case :)